proto/engine/v1/execution_engine.proto

```diff
@@ -157,5 +157,6 @@ message BlobsBundle {
 
 // Blob contains the data that is to be committed on chain.
 message Blob {
-	repeated bytes data = 1 [(ethereum.eth.ext.ssz_size) = "4096,32"];
+	// Each blob consists of `BLS_FIELD_ELEMENT`(32) multiplies `FIELD_ELEMENTS_PER_BLOB`(4096)
+	bytes blob = 1 [(ethereum.eth.ext.ssz_size) = "131072"]; // 32 * 4096 = 131072
```

Comment:
Is there a reason why we are renamed this from `data` back into `blob`?
We originally changed the name in #11601.

Reply:
That was my bad. I fixed it!